feat: added a popup which displays the valid rules of a username while signing up

devArghya-0155 opened the pull request:
fixes #1311 I have added a div which displays the valid rules of a username while signing up. The div will be visible until a correct username is entered. I have also introduced some additional styling during sign up. I am new to open source so please let me know my errors. I will rectify them as soon as possible.

narayan954 reviewed (approved):
looks good to me!

narayan954 commented:
> I have added a div which displays the valid rules of a username while signing up. The div will be visible until a correct username is entered. I have also introduced some additional styling during sign up. I am new to open source so please let me know my errors. I will rectify them as soon as possible.

you did great! :)

devArghya-0155 commented:
> > I have added a div which displays the valid rules of a username while signing up. The div will be visible until a correct username is entered. I have also introduced some additional styling during sign up. I am new to open source so please let me know my errors. I will rectify them as soon as possible.
> 
> you did great! :) By the way, you can link the issues while creating pull requests by mentioning it something like "fixes #issueNumber" so maintainer could directly reference and review the pr faster. I've updated you pr comment to show that.

I will keep this in mind next time!
